Assets/Scripts/SpellTrigger.cs

```diff
@@ -1,17 +1,30 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using UnityEditorInternal;
 using UnityEngine;
 
 public class SpellTrigger : MonoBehaviour
 {
     public GameObject m_Projectile;    
-    public Transform m_SpawnTransform; 
+    public Transform m_SpawnTransform;
+    public IEnumerator coroutine;
 
-    private void Update()
+    private void FixedUpdate()
```

Comment:
You can actually put this back into update now since you have a coroutine controlling firerate